tests/app.config

```diff
@@ -37,11 +37,11 @@
 
             <dependentAssembly>
                 <assemblyIdentity name="Microsoft.Extensions.Logging" publicKeyToken="adb9793829ddae60" culture="neutral"/>
-                <bindingRedirect oldVersion="0.0.0.0-1.1.1.0" newVersion="1.1.1.0"/>
+                <bindingRedirect oldVersion="0.0.0.0-1.1.1.0" newVersion="1.1.0.0"/>
```

Comment:
I think these got added because `Dotnet.Script.NuGetMetadataResolver` depended on 1.1.1. So, it might be that these could just be removed now.